### DISCUSSION TITLE
Only include target file extensions during codegen

### DISCUSSION THREAD

**Discussion by CuiZhicheng** [Netflix/dgs-framework#986] [Q&A]
hi team, can we add a new option as a file extension filter during codegen? 
In our use case, we may add some other files in our schema folder, like reviewer files for github. Currently dgs-codegen just go through all files in the folder, and will fail to parse the content in the reviewer files.

can we add an option to set target file extensions? so that we can filter other files during codegen like this:

https://github.com/Netflix/dgs-codegen/blob/master/graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGen.kt#L96-L98
```
        val schemaFiles = config.schemaFiles
            .flatMap { it.walkTopDown() }
            .filter { it.isFile }
            .filter { includedFileExtensions?.contains(it.extension) ?: true }
```

Thanks!

**Comment by berngp** [comment]
Good idea, will look into it.
